### ISSUE TITLE
Scala native support

### ISSUE THREAD

**Issue by heksesang** [softwaremill/tapir#1821]
It would be nice to have tapir published for scala native with scala 3 (e.g. `tapir-core_native0.4_3`). :)

**Comment by adamw** [comment]
I think we are waiting on scalatest (https://github.com/scalatest/scalatest/issues/2097), see https://github.com/softwaremill/sttp-model/pull/160

**Comment by adamw** [comment]
Hm @Pask423 since you fixed that in the other projects, maybe you could take a look? :)

**Comment by Pask423** [comment]
No problem, I will take a look

**Comment by lolgab** [comment]
I would love to have a tapir frontend for [SNUnit](https://github.com/lolgab/snunit) (a Scala Native HTTP Server library)
